adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -1253,7 +1277,8 @@
 
                     register : function() {
```

Comment:
Just a notion: It is not possible to add app-specific options for registration. Probably because registration is mostly invoked from the login page and not from some application page. Maybe it is possible to somehow get around this for example with some global options specified by a creator of the app. @looorent and @stianst what do you think?